opm/autodiff/BlackoilWellModel_impl.hpp

```diff
@@ -251,7 +373,8 @@ namespace Opm {
     void
     BlackoilWellModel<TypeTag>::
     assemble(const int iterationIdx,
-             const double dt)
+             const double dt,
+             bool onlyDoTheWellTest)
```

Comment:
I could not find where we actually use `onlyDoTheWellTest` as false? If no indeed, maybe we do not need it.

Reply:
onlyDoTheWellTest is false as default 
i.e. 
#382 in BlackoilModelEbos.hpp calls uses it

I added a comment about this in the .hpp file

Reply:
Sorry for the typo. Do we use it as `true` (non-default value?) 

If we do not use any other value than the `default value`, we might not need it.